package_manifest.yaml

```diff
@@ -722,8 +722,6 @@ foreman_client_packages:
         dist: '.el7'
       - name: "foreman-client-{{ foreman_version }}-rhel6"
         dist: '.el6'
-      - name: "foreman-client-{{ foreman_version }}-rhel5"
-        dist: '.el5'
       - name: "foreman-client-{{ foreman_version }}-fedora29"
         dist: '.fc29'
```

Comment:
We also dropped Fedora 29 from 2.2. I think it's safe to also drop it in 2.3 since it's EOL.

Reply:
Will go ahead and drop fc29, I think it just bit me on 2.3 rc1